### PR TITLE
CREST /corporations/npccorps/ and /stations/stationID/ endpoints

### PR DESCRIPTION
**Pull request by jonobrien**
added these two endpoints as they don't seem to be documented yet for CREST.

### CODE REVIEW

**Comment by jonobrien** [comment]
Stations don't seem to have a self-referencing "id" member

**Comment by timpanogos** [comment]
@jonbrien do you follow #devfleet? do you have a user there?

**Comment by jonobrien** [comment]
twitter? yes sometimes I check it.

**Comment by Blacksmoke16** [comment]
Think it might be better to split these into two files.  They don't really have much to do with each other to warrant them being in the same file.

**Comment by jonobrien** [comment]
sounds good

**Comment by Blacksmoke16** [comment]
Maybe even add the station endpoint to the map folder, thinking having location endpoints like /station, or /universe there would make more sense?

**Comment by timpanogos** [comment]
I created a object graph that presents a full RootEndpoint tree instead of the two dimensional flat (and inconsistent) [group][endpoint] structure.  I propose we think about patterning the structure after this tree.

https://github.com/timpanogos/eveonline-third-party-documentation/blob/master/docs/crest/tournament/rootJsonGen.log

**Comment by jonobrien** [comment]
you should probably make an issue to propose changes, or finish your PR and polish things up so people can see the differences.

**Comment by timpanogos** [comment]
I assume xmlapi is intended to slowly trickle over as well right?

**Comment by timpanogos** [comment]
> you should probably make an issue to propose changes, or finish your PR and polish things up so people can see the differences.

yea, I've been in experimental mode for some time now.  I really don't mind going with @CarbonAlabel look and feel with those three or four suggestions I made

I still like the group page per my suggestion, which my current pr reflects, I don't see this changing with a tree ... if you look at that tree every node has that node, then two or so leafs.  I still think your stuff can go in right away as it follows current patterns and we take our time and share finishing up some underconstruction node as we look to consolidate the information into the agreed upon new structure.

**Comment by jonobrien** [comment]
yes, there's several devblogs saying CREST will eventually replace the xmlapi, we'll see how long that takes, soon(tm)

**Comment by OrthoLoess** [comment]
It's also been stated that replicating xml API features is not a current priority. New things come first.

So don't hold your breath 😛

**Comment by jonobrien** [comment]
we don't need to start discussing priorities for features here, CCPlease.

**Comment by timpanogos** [comment]
@jonobrien my only point on the xmlapi was for consideration of documentation's crest main page navigation (which would hopefully take us well into the future)  with the big question being rather to group by functionality or by actual endpoint navigation see my latest activity on #215

File: docs/crest/eve/index.md [hunk 1]
```diff
@@ -14,3 +14,4 @@
 * [Sovereignty Structures and Campaigns](eve_sovereignty.md)
 * [Time](eve_time.md)
 * [Wars and War](eve_wars.md)
+* [NPC Corporations and Stations](eve_npc_corps_and_stations.md)
```
Comment (polarina) [inline docs/crest/eve/index.md]: Can you keep the index sorted alphabetically, please?

**Comment by jonobrien** [comment]
@timpanogos  It is under maps' index file. Unless the recent github review changes are deceiving me. I believe I made that fix already. see 52ab15b

**Comment by jonobrien** [comment]
That would be preferred, but CCP may have other plans, it seems.

**Comment by polarina** [comment]
Thanks! Cherry-picked as 3116b48f1d794af306991815f901ef77d9650e9f.

**Pull request closed without merging**